ui/pages/SignIn.tsx

```diff
@@ -92,7 +92,7 @@ function SignIn({ darkModeEnabled = true }: Props) {
 
   const handleOIDCSubmit = () => {
     const CURRENT_URL = window.origin;
-    return (window.location.href = `/oauth2?return_url=${encodeURIComponent(
+    return (window.location.href = `./oauth2?return_url=${encodeURIComponent(
```

Comment:
This seems to work. Which makes sense, but I thought I read something to the contrary recently -- that while `a`, `img`, `script` etc tags respect `base` tags, the JS world (e.g. setting `location.href`) does not.

However after doing some more testing it seems that it does, and this works fine, even if you end up at localhost:9001/wego/signin/ (with a trailing slash), which still routes correctly, and then logs into OIDC correctly too..

Reply:
JSDOM etc at least don't support navigation so this is a trickier case to test w/out more involved acceptance tests..